### PR TITLE
react ui has been added

### PR DESCRIPTION
**Pull request by Bhanu707**

### CODE REVIEW

File: ui/.gitignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.DS_Store
```
Comment (satyapriyamishra222) [inline ui/.gitignore]: Please remove this

File: ui/README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```
Comment (satyapriyamishra222) [inline ui/README.md]: I believe we can get rid of this file. Let's have a project centric readme.md file or at least change the text.

File: ui/public/index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
+  </head>
+  <body>
+    <noscript>You need to enable JavaScript to run this app.</noscript>
+    <div id="root"></div>
+    <!--
```
Comment (satyapriyamishra222) [inline ui/public/index.html]: Pls remove the commented code in this file as we don't need them in future.

File: ui/src/App.css [hunk 4]
```diff
@@ -0,0 +1,38 @@
+.App {
```
Comment (satyapriyamishra222) [inline ui/src/App.css]: Pls remove the styles completely. We will have to write them separately.

File: ui/src/App.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import logo from './logo.svg';
```
Comment (satyapriyamishra222) [inline ui/src/App.js]: Unused import. Pls remove it.

File: ui/src/index.css [hunk 6]
```diff
@@ -0,0 +1,13 @@
+body {
```
Comment (satyapriyamishra222) [inline ui/src/index.css]: No need of these styles.

File: ui/src/Home/index.js [hunk 7]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+
+class Home extends React.Component {
```
Comment (satyapriyamishra222) [inline ui/src/Home/index.js]: We will be working on functional components. No class based components please.

File: ui/src/index.js [hunk 8]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+
+ReactDOM.render(
+  <React.StrictMode>
```
Comment (satyapriyamishra222) [inline ui/src/index.js:8]: Please wrap this portion in an error boundary.

File: ui/src/logo.svg [hunk 9]
```diff
@@ -0,0 +1 @@
...21.2 430c-9.3-9.6-18.6-20.3-27.8-32 9 .4 18.2.7 27.5.7 9.4 0 18.7-.2 27.8-.7-9 11.7-18.3 22.4-27.5 32zm-74.4-58.9c-14.2-2.1-27.9-4.7-41-7.9 3.7-12.9 8.3-26.2 13.5-39.5 4.1 8 8.4 16 13.1 24 4.7 8 9.5 15.8 14.4 23.4zM420.7 163c9.3 9.6 18.6 20.3 27.8 32-9-.4-18.2-.7-27.5-.7-9.4 0-18.7.2-27.8.7 9-11.7 18.3-22.4 27.5-32zm-74 58.9c-4.9 7.7-9.8 15.6-14.4 23.7-4.6 8-8.9 16-13 24-5.4-13.4-10-26.8-13.8-39.8 13.1-3.1 26.9-5.8 41.2-7.9zm-90.5 125.2c-35.4-15.1-58.3-34.9-58.3-50.6 0-15.7 22.9-35.6 58.3-50.6 8.6-3.7 18-7 27.7-10.1 5.7 19.6 13.2 40 22.5 60.9-9.2 20.8-16.6 41.1-22.2 60.6-9.9-3.1-19.3-6.5-28-10.2zM310 490c-13.6-7.8-19.5-37.5-14.9-75.7 1.1-9.4 2.9-19.3 5.1-29.4 19.6 4.8 41 8.5 63.5 10.9 13.5 18.5 27.5 35.3 41.6 50-32.6 30.3-63.2 46.9-84 46.9-4.5-.1-8.3-1-11.3-2.7zm237.2-76.2c4.7 38.2-1.1 67.9-14.6 75.8-3 1.8-6.9 2.6-11.5 2.6-20.7 0-51.4-16.5-84-46.6 14-14.7 28-31.4 41.3-49.9 22.6-2.4 44-6.1 63.6-11 2.3 10.1 4.1 19.8 5.2 29.1zm38.5-66.7c-8.6 3.7-18 7-27.7 10.1-5.7-19.6-13.2-40-22.5-60.9 9.2-20.8 16.6-41.1 22.2-60.6 9.9 3.1 19.3 6.5 28.1 10.2 35.4 15.1 58.3 34.9 58.3 50.6-.1 15.7-23 35.6-58.4 50.6zM320.8 78.4z"/><circle cx="420.9" cy="296.5" r="45.7"/><path d="M520.5 78.1z"/></g></svg>
```
Comment (satyapriyamishra222) [inline ui/src/logo.svg]: Pls remove this file entirely.

File: ui/.gitignore (hunk 1 above)
Comment (Bhanu707) [reply ui/.gitignore]: removed

**Review by satyapriyamishra222 — APPROVED** [review]

**Pull request merged**